pkg/cvo/sync_worker.go

```diff
@@ -751,12 +751,44 @@ func summarizeTaskGraphErrors(errs []error) error {
 	if len(errs) == 1 {
 		return errs[0]
 	}
+	// hide the generic "not available yet" when there are more specific errors present
+	if filtered := filterErrors(errs, isClusterOperatorNotAvailable); len(filtered) > 0 {
+		return newMultipleError(filtered)
+	}
+	// if we're only waiting for operators, condense the error down to a singleton
 	if err := newClusterOperatorsNotAvailable(errs); err != nil {
 		return err
 	}
 	return newMultipleError(errs)
```

Comment:
This is the only `newMultipleErrors` consumer.  It feels strange to have some preprocessing locally; can we add this new logic inside `newMultipleErrors` (and drop the local collapse, since `newMultipleErrors` already does that internally)?

Reply:
It’s three different flows - I expect there to be more in the future.  MultipleErrors is for when we don’t know anything